st-thomas-more/stm-dashboard
# 5 add profiles

To test:

Depends on #4

Follow instructions in README. Make sure you create the .env file. Start the app. Make sure you run it on port 3000 and visit it at localhost:3000. When you log in with your Google account it should show some profile information on the homepage. Also run `npm test` to verify the few tests we have so far work.
